app/src/main/res/layout/share_episode_dialog.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:padding="16dp">
+
+    <RadioGroup
+        android:id="@+id/share_dialog_radio_group"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:orientation="vertical">
+
+        <RadioButton android:id="@+id/share_episode_website_radio"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/share_dialog_episode_website_label"
+            android:checked="true"
+            />
+        <RadioButton android:id="@+id/share_media_file_radio"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/share_dialog_media_file_label"
+            />
+    </RadioGroup>
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="@string/share_dialog_include_label"/>
+
+    <Switch
+        android:id="@+id/share_start_at_timer_dialog"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/share_playback_position_dialog_label" />
```

Comment:
I think the dialog could look more balanced if this was a CheckBox instead of a Switch. Then all the blue elements line up on the left. I am not completely sure, though. What do you think?

Reply:
Yes I agree, I will try to update it.